data/World/Shadow Temple.json

```diff
@@ -30,7 +30,13 @@
             "Shadow Temple Early Silver Rupee Chest": "True"
         },
         "exits": {
-            "Shadow Temple Huge Pit": "has_explosives and (Small_Key_Shadow_Temple, 1)"
+            # If the shortcut is open, reverse shadow becomes an option, so we need to check for 4 keys.
+            # We don't need to check for 5 since if they can use 4 and the shortcut is open then the last center door is irrelevant.
```

Comment:
Is it actually irrelevant or is it possible to use up the 4 keys on the connecting doors and become softlocked?

Reply:
I've worked through it thoroughly, it's irrelevant.

The 4th central door is just for boat access, which the shortcut already provides.

Reply:
I don't believe it is irrelevant. I explain in a comment what I think the problem is.